### PR TITLE
refactor:sso home/index page. resolved #978

### LINKED ISSUE #978: SSO中Home/Index 页面根据Figma调整

### PR DESCRIPTION
**Pull request by MapleWithoutWords**
image

### CODE REVIEW

**Comment by MapleWithoutWords** [comment]
resolved #978

File: src/Web/Masa.Auth.Web.Sso/Pages/Home/Index.razor [hunk 1]
```diff
@@ -1,19 +1,43 @@
...
+                @T("GoToAuthorization")
+            </MButton>
+        </div>
+    </MCardText>
+</MCard>
 
+@code {
+
+    private void Logout()
+    {
```
Comment (MayueCif) [inline src/Web/Masa.Auth.Web.Sso/Pages/Home/Index.razor:41]: ?

File: src/Web/Masa.Auth.Web.Sso/Pages/Home/Index.razor [hunk 2]
```diff
@@ -1,19 +1,43 @@
...
+            <li class="mt-7">
+                Click <a href="/grants">here</a> to manage your stored grants.
+            </li>
+            <li class="mt-7">
+                Click <a href="/ciba/all">here</a> to view your pending CIBA login requests.
+            </li>
+        </ul>
+        <div class="mt-12 d-flex flex-wrap justify-center">
+            <MButton Class="primary rounded-pill" Width="340" Height="48" OnClick="Logout">@T("LogoutOfLogin")</MButton>
+            <MButton Class="rounded-pill mt-6 mb-10" Outlined Color="#4318FF" Width="340" Height="48">
```
Comment (MayueCif) [inline src/Web/Masa.Auth.Web.Sso/Pages/Home/Index.razor:31]: delete

File: src/Web/Masa.Auth.Web.Sso/Pages/Home/Index.razor [hunk 3]
```diff
@@ -1,19 +1,43 @@
...
-    <li>
-        Click <a href="/ciba/all">here</a> to view your pending CIBA login requests.
-    </li>
-</ul>
+<MCard MaxWidth="512">
+    <MCardTitle Class="h5" Style="color: #1B2559;">
+        Masa Sso
+    </MCardTitle>
+    <MCardSubtitle Class="grey--text" Style="color: #7681AB !important;line-height: 24px;font-size: 16px;">
+        @T("You agree to MASA technical team to obtain your following rights regarding Tencent wechat")
```
Comment (MayueCif) [inline src/Web/Masa.Auth.Web.Sso/Pages/Home/Index.razor:10]: Welcome to Masa Stack SSO

**Review by MayueCif — COMMENTED** [review]
@MapleWithoutWords

**Review by MayueCif — APPROVED** [review]

**Pull request merged**